tests/test_association_db_sqlite.py

```diff
@@ -0,0 +1,266 @@
+#
+# LSST Data Management System
+# Copyright 2017 LSST/AURA.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+from __future__ import absolute_import, division, print_function
+
+import numpy as np
+import unittest
+
+from lsst.ap.association import *
+from lsst.afw.coord import Coord
+import lsst.afw.geom as afwGeom
+import lsst.utils.tests
+from test_dia_collection import create_test_dia_objects
+from test_dia_object import create_test_dia_sources
+
+
+class TestAssociationDBSqlite(unittest.TestCase):
+
+    def setup(self):
```

Comment:
I don't really want empty methods here. They serve no purpose other than adding cruft, and this one in particular is no good because it should be `setUp()` not `setup()`.